src/Employer/Employer.Web/ViewModels/Part1/Employer/EmployerViewModel.cs

```diff
@@ -17,6 +17,10 @@ public class EmployerViewModel : EmployerEditModel
             nameof(EmployerEditModel.AddressLine4),
             nameof(EmployerEditModel.Postcode)
         };
+
+        public bool IsWizard { get; set; }
```

Comment:
Could you put these properties in an abstract class that the part 1 viewmodel inherit from

Reply:
Unfortunately no because they already inherit from their respective Edit Models